plugins/entando-plugin-jpkiebpm/src/main/java/org/entando/entando/plugins/jpkiebpm/aps/internalservlet/BpmTypeTaskFormAction.java

```diff
@@ -142,7 +154,8 @@ public String view() {
     }
 
     public String render(DataObject dataobject, String contentModel, String langCode, RequestContext reqCtx) {
-        String renderedEntity = null;
+        dataobject.getAttributeList().forEach(f -> logger.info("render {} value {} type {}", f.getName(), f.getValue(), f.getType()));
```

Comment:
This statement should be debug level. And since the loop is outside of the debu this needs to be wrapped in an if(logger.isDebugEnabled()) check